03/app.js

```diff
@@ -0,0 +1,37 @@
+const fileInput = document.querySelector('input')
+
+const showFileData = function (e) {
+	const file = e.target.files[0]
+	const imgList = document.querySelector('.images-list')
+	const imgListItem = document.querySelector('.images-list__item.images-list__item--prototype')
+
+	if (file && file.type.includes('image')) {
+		const reader = new FileReader()
+		let listItemClone = imgListItem.cloneNode(true)
+
+		const fileName = file.name
+		const fileSize = file.size
+		const fileSizeInMB = (fileSize / (1024 * 1024)).toFixed(2)
+        
+		const addAnotherGraphicFile = function () {
```

Comment:
Staraj się tworzyć funkcje poza miejscem jej wywołania i przekazuj odpowiednie dane przez parametry - to jest najlepsza praktyka. W tym wypadku wyglądałoby to mniej więcej tak:
```
if(...) {
    // ...
    
    addAnotherGraphicFile(fileInput.files, listItemClone, reader);
}

function addAnotherGraphicFile(files, listItemPrototype, reader) { // normalna funkcja, aby działał hoisting
// ciało funkcji z odpowiedno podmienionymi nazwami zmiennych
}
```

Reply:
tu też poprawione